wcomponents-core/src/main/java/com/github/bordertech/wcomponents/util/mock/MockRequest.java

```diff
@@ -97,7 +99,6 @@ public void addParameterForButton(final UIContext uic, final WButton button) {
 	/**
 	 * Sets mock file upload contents.
 	 *
-	 * @deprecated Because it appears unused
```

Comment:
We cant deprecate this based on our usage. As it is public it maybe used by projects using `WComponents`.

Reply:
Originally this method had a `@deprecated` in it's javadoc, I have removed it.

Reply:
ooops...You deleted it. Good job.